www/src/data/sidebars/doc-links.yaml

```diff
@@ -32,8 +32,10 @@
           link: /docs/deploying-to-now/
         - title: Deploying to GitLab Pages
           link: /docs/deploying-to-gitlab-pages/
-        - title: Hosting on Netlify
+        - title: Deploying to Netlify
```

Comment:
if we change the title here, we would need to change it in the doc, the URL, and also create a redirect. It seems like the doc is about hosting though. Am I wrong about that?

Reply:
@shannonbux I'm not sure there's a meaningful difference--and this seems more consistent (e.g. all deploying guides start with `Deploying to-`? I can revert this change, though.)

Reply:
Oh _whoops_ we actually do have two separate docs here.

[Deploying to Netlify](https://www.gatsbyjs.org/docs/deploying-to-netlify/)
[Hosting on Netlify](https://www.gatsbyjs.org/docs/hosting-on-netlify/)

... should we? It seems like the Hosting on Netlify document is a little more fully featured.

Reply:
actually - let's just side step this. I'll revert this change.